tests/RFC3339Test.elm

```diff
@@ -345,6 +345,48 @@ decodeTimestampTest =
         )
 
 
+isLeapyearTest : Test
+isLeapyearTest =
+    describe "correctly determines if a year is a isLeapyearTest"
+        [ test "with leapyears"
+            (\_ ->
+                [ 1904
+                , 2040
+                , 2140
+                , 1944
+                , 2044
+                , 2144
+                , 1948
+                , 2048
+                , 2148
+                , 1952
+                , 2052
+                , 2068
+                , 2168
+                , 1972
+                , 2072
+                , 2172
+                , 1976
+                , 2076
+                , 2176
+                , 2092
+                , 2192
+                , 1996
+                , 2096
+                , 2196
+                ]
```

Comment:
Do we want to add some end-of-century years to the tests?

```suggestion
                , 2096
                , 2196
                , 4000
                , 8000
                ]
```

Reply:
Good idea, done!